Server/64base.py

```diff
@@ -0,0 +1,65 @@
+import pymodm
+from pymodm import connect
+from flask import Flask, jsonify, request
+from flask_cors import CORS
+#import database_functions
+#import models
+import datetime
+from skimage import exposure
+import base64
+import numpy as np
+import uuid
+import os
+import math
+from skimage import util
+import  PIL
+app = Flask(__name__)
+CORS(app)
+connect("mongodb://vcm-3572.vm.duke.edu:27017/imaging_processing")  # open up connection to db  ????
+
+
+@app.route("/api/histogram_equalization", methods=["POST"])
+def store_heart_rate():
+    """
+        send back the histogram equalization processing result
```

Comment:
fix formatting on all of these docstrings -> all of the beginnings of the lines should align

Reply:
also, these docstrings need to include what the function is taking in, what it returns, and any errors that are thrown